### PR TITLE
Use pathlib with Country graph.

### PR DESCRIPTION
**Pull request by miskosz**
Polishing the code before making bigger changes.

### CODE REVIEW

File: covid_graphs/covid_graphs/country_report.py [hunk 1]
```diff
@@ -1,24 +1,36 @@
...
-            self.daily_recovered = np.array([day.recovered for day in country_data.stats])
-            self.daily_active = daily_positive - self.daily_recovered - self.daily_dead
-            self.date_list = [
-                f"{day.date.year}-{day.date.month:02d}-{day.date.day:02d}"
-                for day in country_data.stats
-            ]
+    def __init__(self, data_dir: Path, country: Country):
+        """Constructs a numpy representation of data read from 'data_dir' a given country."""
+        country_data = CountryData()
+        data_file = Path(data_dir / f'{country.name}.data')
```
Comment (lukipuki) [inline covid_graphs/covid_graphs/country_report.py]: Doesn't the file need to be closed?

**Review by lukipuki — APPROVED** [review]

File: covid_graphs/covid_graphs/country_report.py (hunk 1 above)
Comment (miskosz) [reply covid_graphs/covid_graphs/country_report.py]: This does not open the file, just creates a path object (essentially just a smarter string). Will rename the variable for clarity.

Comment (miskosz) [reply covid_graphs/covid_graphs/country_report.py]: For completeness, from https://docs.python.org/3/library/pathlib.html:
```
Path.read_text(encoding=None, errors=None)
...
The file is opened and then closed.
```

Comment (lukipuki) [reply covid_graphs/covid_graphs/country_report.py]: Smart!

**Pull request merged**